### PR TITLE
Added API endpoint for pushnotification categories

### PR DESCRIPTION
**Pull request by KiOui**
Added API endpoint for pushnotification categories.

Testing:

Go to `/api/docs`, select the v2 api docs and see if the endpoint works.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,6 @@ result-*
 # direnv
 .envrc
 .env*
+
+# macOS
+.DS_Store
```
Comment (JobDoesburg) [inline .gitignore:42]: Let's do this in a separate PR

**Review by JobDoesburg — CHANGES_REQUESTED** [review]

File: website/pushnotifications/api/v2/views.py [hunk 2]
```diff
@@ -74,6 +74,16 @@ def perform_update(self, serializer):
         serializer.save(user=self.request.user)
 
 
+class CategoryListView(ListAPIView):
+    """Returns an overview of all available categories for push notifications."""
+
+    serializer_class = CategorySerializer
+    queryset = Category.objects.all()
+    required_scopes_per_method = {
```
Comment (JobDoesburg) [inline website/pushnotifications/api/v2/views.py:82]: This does not need to be per method

**Review by JobDoesburg — APPROVED** [review]

**Review by se-bastiaan — APPROVED** [review]
